box.json

```diff
@@ -0,0 +1,20 @@
+{
+  "alias": "coveralls.phar",
+  "chmod": "0755",
+  "directories": ["src"],
+  "extract": true,
+  "files": [
+    "LICENSE"
+  ],
+  "finder": [
+    {
+      "name": ["*.php", "*.pem"],
+      "exclude": ["tests"],
+      "in": "vendor"
```

Comment:
:facepalm:
no need to use both finder and directories... you have two dirs you want to use and use and in `finder` and second in `directories`. Finder would be enought ;)